### DISCUSSION TITLE
自从把“安卓系统”加进分应用代理，一切变得流畅

### DISCUSSION THREAD

**Discussion by wfyanmnm** [2dust/v2rayNG#2483] [Ideas]
![Screenshot_20230823_175908.jpg](https://github.com/2dust/v2rayNG/assets/1192270/3d4bcfe1-720b-4366-b3d3-ca774c2189a8)

之前总有些地方卡住不动，用抓包工具抓出来发现是安卓系统在请求，把它加进分应用代理，一切都好了。

**Comment by 1315aaa** [comment]
确实有改变，具体是什么原因啊？

**Reply by Hermit-yu** [reply]
应该是底层请求方面的问题

**Reply by zazitufu** [reply]
分应用代理之后，比如这是个浏览器，那它是强制全部走代理吗？上一层设置里面的“绕过国内和局域网”还会生效么？
